cypress/page-objects/todo.page.js

```diff
@@ -0,0 +1,53 @@
+export class TodoPage {
+    header () { return cy.get('h1')}
+    newTodo() { return cy.get('.new-todo')}
+    todoItems () { return cy.get('.todo-list li')}
+    checkbox() {return cy.get('.toggle[type="checkbox"]')}
+
+    addTodos(items){
+        for (let item of items) {
+            this.newTodo().type(item).type('{enter}')
+        }}
+
+    markTodos(){
```

Comment:
This name is not descriptive enough. There could be different types of "marking". In our case, "mark as completed". So, `markCompletedTodos` will fit the purpose.